util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialsConfiguration.java

```diff
@@ -137,6 +137,10 @@ public class HadoopCredentialsConfiguration {
   public static final HadoopConfigurationProperty<RedactedString> PROXY_PASSWORD_SUFFIX =
       new HadoopConfigurationProperty<>(".proxy.password");
 
+  /** Key suffix for setting the read timeout (in milliseconds) for HTTP request. */
+  public static final HadoopConfigurationProperty<Integer> READ_TIMEOUT_SUFFIX =
+      new HadoopConfigurationProperty<>(".http.read-timeout", 20 * 1000);
```

Comment:
Seems like we have this config property declared 2 times now?

Reply:
Thanks for catching this. I removed the duplicated definition under `/gcs`.